Gordon360/Models/ViewModels/EmailAccountViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Gordon360.Models.ViewModels
+{
+    public class EmailAccountViewModel
```

Comment:
Is this used anywhere in your code? It doesn't seem like it. Also, if it is, we should probably refactor it so that this isn't necessary.

Reply:
It's used as the getter of the email credentials of 360train email acc from appsettings

Reply:
yep removed it after reading the comments below